components/homme/test/reg_test/run_tests/create-theta-temporary.sh

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Curious: where is this file used?

Reply:
This was committed by a mistake, I was going to remove it in the last round of cleaning. This file is created temporarily by make when re-making a list of theta-l xx tests. However, that set of scripts is not flexible enough and needs updating (maybe in another PR).